opentelemetry-sdk/tests/metrics/test_metric_cardinality_limit.py

```diff
@@ -0,0 +1,40 @@
+
+from logging import WARNING
+from unittest import TestCase
+
+from opentelemetry.sdk.metrics import MeterProvider
+
+from opentelemetry.sdk.metrics.export import (
+    InMemoryMetricExporter,
+    PeriodicExportingMetricReader,
+)
+
+class TestMetricCardinalityLimit(TestCase):
+
+    def setUp(self):
+        self.exporter = InMemoryMetricExporter()
+        self.reader = PeriodicExportingMetricReader(exporter=self.exporter, export_interval_millis=500)
```

Comment:
From Srikanth's comment on the other PR, can we just use `InMemoryMetricReader` instead? Here is an example from another test https://github.com/open-telemetry/opentelemetry-python/blob/91ac1bea6ed2b50e7e17b91cec2770674f869517/opentelemetry-sdk/tests/metrics/integration_test/test_histogram_export.py#L25C9-L39